saviSimulation/savi/jason_processing/SAVIWorld_model.java

```diff
@@ -196,28 +210,28 @@ public void setup() {
 		// ======= set up Jason BDI agents ================
 		Map<String, AgentModel> agentList = new HashMap<String, AgentModel>();
 
-		for (WorldObject wo : objects) {// Create UAS agents
-			if (wo instanceof UAS) {
-				agentList.put(((UAS) wo).getBehavior().getID(), ((UAS) wo).getBehavior());
+		for (WorldObject wo : objects) {// Create UxV agents
+		if(wo instanceof UxV) {
+			agentList.put(((UxV)wo).getBehavior().getID(), ((UxV)wo).getBehavior());
```

Comment:
Maybe the agentList could be populated at the same time as the UAV/UGV are created, as with the wifiParticipants list? (this also requires moving line 211)

Reply:
Fixed by @cruizm